fetcher.py

```diff
@@ -0,0 +1,99 @@
+    "csse_covid_19_data/csse_covid_19_time_series"
+)
+FILENAME_JOHN_HOPKINS = {
+    "confirmed": "time_series_19-covid-Confirmed.csv",
+    "death": "time_series_19-covid-Deaths.csv",
+    "recovered": "time_series_19-covid-Recovered.csv",
+}
+
+URL_COUNTRY_ISO = (
+    "https://gist.githubusercontent.com/tadast/8827699/raw/"
+    "7255fdfbf292c592b75cf5f7a19c16ea59735f74/"
+    "countries_codes_and_coordinates.csv"
+)
+
+MAP_UNMATCHED_COUNTRIES = {
+    'Congo (Kinshasa)': "Congo, the Democratic Republic of the",
+    "Cote d'Ivoire": "Côte d'Ivoire",
+    'Czechia': "Czech Republic",
+    'Eswatini': "Swaziland",
+    'Holy See': "Italy",
+    'Iran': "Iran, Islamic Republic of",
+    'Korea, South': "South Korea",
+    'Moldova': "Moldova, Republic of",
+    'North Macedonia': "Macedonia, the former Yugoslav Republic of",
+    'Reunion': "Réunion",
+    'Taiwan*': "Taiwan",
+    'US': "United States",
+    'occupied Palestinian territory': "Palestinian Territory, Occupied",
+}
+
+MISSING_COUNTRIES = pd.DataFrame({
+    "Country": ["Curacao"],
+    "Alpha-2 code": ["CW"],
+    "Alpha-3 code": ["CUW"],
+    "Numeric code": [531.0],
+    "Latitude (average)": [12.169570],
+    "Longitude (average)": [-68.990021],
+})
+
+UNMATCHED_COUNTRIES = ['Cruise Ship']
+
+
+def fetch_john_hopkins_data():
+    df_covid = {
+        key: pd.read_csv(os.path.join(URL_BASE, filename))
+        for key, filename in FILENAME_JOHN_HOPKINS.items()
+    }
+    columns_drop = ["Province/State", "Lat", "Long"]
+    for key in df_covid:
+        df_covid[key] = df_covid[key].drop(columns=columns_drop)
+        df_covid[key] = df_covid[key].groupby("Country/Region").sum()
+        df_covid[key] = df_covid[key].stack().to_frame().reset_index().rename(
+            columns={"level_1": "date", 0: "cases"}
+        )
+        df_covid[key]["date"] = df_covid[key]["date"].apply(
+            lambda x: pd.to_datetime(x)
+        )
+        df_covid[key]["type"] = key
+    df_covid = pd.concat(df_covid.values(), ignore_index=True)
+    df_covid = df_covid.replace({"Country/Region": MAP_UNMATCHED_COUNTRIES})
+
+    df_countries = pd.read_csv(URL_COUNTRY_ISO)
+    for col_name in df_countries.columns:
+        df_countries[col_name] = df_countries[col_name].str.strip('. "."')
+    float_column = [
+        "Numeric code", "Latitude (average)", "Longitude (average)"
+    ]
+    df_countries[float_column] = df_countries[float_column].astype(float)
+    df_countries = pd.concat(
+        [df_countries, MISSING_COUNTRIES], ignore_index=True
+    )
+
+    df = pd.merge(
+        left=df_covid,
+        right=df_countries,
+        left_on="Country/Region",
+        right_on="Country",
+        how="inner",
+    )
+
+    assert df["Country"].unique().shape == (141,), \
+        "Missing countries when making the merge"
+
+    df = df.rename(
+        columns={
+            "Country/Region": "name",
+            "Country": "country_region",
+            "Alpha-3 code": "alpha-3",
```

Comment:
We're renaming "alpha-3" to "iso" in the input_data function. Maybe we could change this right here.